api/proto/dop/issue/sync/sync.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package erda.dop.issue.sync;
+option go_package = "github.com/erda-project/erda-proto-go/dop/issue/sync/pb";
+import "google/api/annotations.proto";
+import "common/openapi.proto";
+
+service IssueSyncService {
+  option(erda.common.openapi_service) = {
+    service : "dop",
+    auth : {
+      check_login : true,
+      check_token: true,
+    }
+  };
+
+  rpc IssueSync(IssueSyncRequest) returns(IssueSyncResponse) {
+    option(google.api.http) = {
+      post : "/api/issues/actions/sync",
+    };
+    option(erda.common.openapi) = {
+      path : "/api/issues/actions/sync",
+    };
+  }
+}
+
+message IssueSyncRequest {
+    int64 parentID = 1;
+    int64 iterationID = 2;
+    List addition = 3;
+    List deletion = 4;
```

Comment:
need an option to indicate whether sync child or not?

Reply:
no. if we are ok with https://github.com/erda-project/erda/pull/4522#discussion_r843562270